tests/test_engine.py

```diff
@@ -72,6 +79,27 @@ def test_file_tests():
     os.unlink(output)
 
 
+def test_handlebars_file_tests():
+    output = "test.txt"
+    path = os.path.join("tests", "fixtures", "handlebars_tests")
+    engine = EngineHandlebars([path], path)
+    engine.render_to_file("file_tests.template", "file_tests.json", output)
+    with open(output, "r") as output_file:
+        content = output_file.read()
+        eq_(content, "here")
+    os.unlink(output)
+
+
```

Comment:
in here, you can use: @raises(FileNotFoundError)

raises can be imported from nose.tools